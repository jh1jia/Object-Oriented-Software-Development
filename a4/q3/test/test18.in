test16.txt dropfirst 0 count 2 count 9 count 1
