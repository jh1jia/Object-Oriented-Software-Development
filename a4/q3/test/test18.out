12345678910
11234567812130
3333333333
01415163456782
3876543172180
7861904203364
21
224
3
4
5
6
7
0
5
